example.com.ssl.conf.erb

```diff
@@ -0,0 +1,30 @@
+server {
+    listen 443 ssl;
```

Comment:
This is more configuration than is necessary.

For prod, our conf file is written from a template and the domain being used gets written.

https://github.com/openpredictionmarkets/socialpredict/blob/main/data/nginx/ssl_vhosts/app.conf.template

Reply:
example.com.ssl.conf.erb is template of https-portal